CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# Contribuir
+
+## Introdução
+Muito obrigado por considerar uma contribuição ao site do Teresina Hacker Clube :tada:
+Por conta de pessoas como você que a comunidade de open source é tão bacana!
+
+Você pode contribuir com:
+* Reportando bugs no nosso site
+* Escrevendo artigos para o nosso site
+* Entrando em contato para promover eventos
+* Abrindo **Pull Requests** através de (issues)[https://github.com/teresinahc/teresinahc.github.io/issues] abertas em nosso repositório
+
+## Seu primeiro Pull Request (PR)
+
+Seu primeiro PR? Fique tranquilo, aqui tem um tutorial do próprio (Github)[https://firstcontributions.github.io/] que vai te ajudar nesses primeiros passos. Escolha uma issue, nos comunique que está trabalhando nela e seja feliz com sua contribuição :clap:
+
+## Submetendo código
+Qualquer mudança de código deve ser enviada através de Pull Request. Faça uma descrição bacana e siga boas práticas de código e commit para ter uma revisão rápida e aprovada. Não sinta medo ao enviar a alteração, nós iremos discutir sobre ela caso seja necessário uma melhoria. Caso esteja trabalhando ainda nela, adicione a tag WIP (_Work in Progress_).
```

Comment:
Qualquer mudança de código deve ser enviada através de Pull Request. Escreva uma descrição objetiva e clara, e siga boas práticas de código e commit para ter uma revisão rápida e aprovada. Não sinta medo ao enviar a alteração, nós iremos discutir sobre ela caso seja necessária alguma melhoria. Caso esteja trabalhando ainda nela, adicione a tag WIP: <Nome do PR>. WIP significa "Work in Progress", em português, trabalho em progresso.